modules/spring-config/src/main/resources/META-INF/mule.xsd

```diff
@@ -4075,7 +4075,7 @@
                 <xsd:attribute name="filterOnErrorType" type="errorIdentifierType" use="optional">
                     <xsd:annotation>
                         <xsd:documentation>
-                            The full name of the error types or a comma separated list of full names, to filter out of the resulting payload when it occurs. This is useful to use validations inside this component.
+                            A comma separated list of error types that should be handled by dropping the split part. These errors are handled so that items that cause them when being processed won't be included in the aggregated response collection, rather than propagating the error. This is useful to use validations inside this component.
```

Comment:
Still not that clear... 
`A comma separated list of error types that should be handled by filtering the split part, rather than propagating the error.  Filtered parts are not included in the aggregated response collection. This is useful to use validations inside this component.`